pkg/sessionctx/variable/tidb_vars.go

```diff
@@ -1465,6 +1467,7 @@ const (
 	DefTiDBOptObjective                               = OptObjectiveModerate
 	DefTiDBSchemaVersionCacheLimit                    = 16
 	DefTiDBIdleTransactionTimeout                     = 0
+	DefEnableParallelSort                             = false
```

Comment:
Just a small question, why we don't enable it by default?

Reply:
> Just a small question, why we don't enable it by default?

Because we want to simultaneously enable the parallel sort and parallel sort spill in v8.1 .